curriculum/challenges/english/07-scientific-computing-with-python/learn-encapsulation-by-building-a-projectile-trajectory-calculator/6633c06601c081735063b528.md

```diff
@@ -0,0 +1,303 @@
+---
+id: 6633c06601c081735063b528
+title: Step 17
+challengeType: 20
+dashedName: step-17
+---
+
+# --description--
+
+You have built a projectile trajectory calculator.
+
```

Comment:
It's pretty straightforward, but I'd specify that this should be outside the classes